schemas/fields/urlField.ts

```diff
@@ -4,9 +4,18 @@ import ResourceUrlInput from "../../components/ResourceUrlInput";
 const urlField = defineField({
   title: "URL",
   description:
-    "This should be a valid web address, usually starting with 'https://'",
+    "This should be a valid web address, usually starting with 'https://'. Paths may be used but be aware this likely points to a specific resource, rather than a website overall.",
   name: "resourceUrl",
   type: "url",
+  validation: (Rule) =>
+    Rule.custom((resourceUrl) => {
```

Comment:
Custom validators run against undefined values, so you need to check for undefined and return early if it is. Something like:
```suggestion
    Rule.custom((resourceUrl) => {
    if (typeof resourceUrl === 'undefined') {
      return true
    }
```
...otherwise the function may crash

Reply:
Done